xarray/core/nanops.py

```diff
@@ -184,9 +184,9 @@ def nanvar(a, axis=None, dtype=None, out=None, ddof=0):
         a, axis=axis, dtype=dtype, ddof=ddof)
 
 
-def nanstd(a, axis=None, dtype=None, out=None):
+def nanstd(a, axis=None, dtype=None, out=None, ddof=0):
     return _dask_or_eager_func('nanstd', eager_module=nputils)(
-        a, axis=axis, dtype=dtype)
+        a, axis=axis, dtype=dtype, ddof=ddof)
```

Comment:
Hi,
Thanks for fixing.
Is there any reason why not all remaining keywords are passed on (as **kwds or something)?
I think that would help to maintain numpy compatibility.
Cheers.

Reply:
This sounds a good idea.
I just did not notice that there are significant keywords other than `axis` and `out`. 
Your contribution is welcome ;)